app/controllers/house_members_favorites_controller.rb

```diff
@@ -0,0 +1,30 @@
+class HouseMembersFavoritesController < ApplicationController
+  def create
+    if !current_user.nil?
+      create_house_favorite(params[:id])
+      flash[:notice] = "Successfully added to favorites"
+      redirect_to house_member_path(params[:id])
+    else
+      flash.now[:alert] = "You must sign in to favorite a member"
+    end
+  end
+
+  def destroy
+    delete_house_favorite(params[:id])
+    flash[:notice] = "Successfully removed from favorites"
+    redirect_to house_member_path(params[:id])
+  end
+
+  private
+
+  def create_house_favorite(id)
+    current_user.house_favorites.create(
+      house_member_id: id
+    )
+  end
```

Comment:
Nice job on creating helper methods, but almost wondering if these couldn't be put on the user model instead of in the controller though. Otherwise looks good!